src/Portfolio/Card.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./Card.scss";
+
+const Card = ({ image, imageName, name, desc }) => {
+  return (
+    <div className="card">
+      <img src={image} alt={imageName} className="card-image" />
+      <div className="card-titlebox">
+        {name}
+        <p className="card-paragraph">{desc}</p>
```

Comment:
This structure is a bit confusing. You have a title component but its also got a description in it and the title itself isn't wrapped in an element.

I would do a couple of things here. Wrap the {name} in a h2/h3 to make it semantic and seperate the heading from the description. Then maybe also rename the outer div class. I get what you mean by titlebox but its confusing because you have an actual title in that component (which is something different)